### PR TITLE
.github/stale.yml: configure probot/stale to automatically close stale issues

### PR DESCRIPTION
**Pull request by liewegas**
Initially only warn about closing issues, but do not close them.

Signed-off-by: Sage Weil <sage@redhat.com>

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+#daysUntilClose: 7
+
+# for now, don't close anything by setting this limit very high:
+daysUntilClose: 700
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - pinned
+  - security
+  - "[Status] Maybe Later"
```
Comment (branch-predictor) [inline .github/stale.yml]: Hopefully it won't end up with 752 PRs labeled "Maybe Later". :)

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: true
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: true
+
+# Label to use when marking as stale
+staleLabel: stale
```
Comment (ErwanAliasr1) [inline .github/stale.yml]: Semantic question. Shouldn't it be "staled" rather than "stale" ?

**Comment by liewegas** [comment]
retest this please

File: .github/stale.yml (hunk 2 above)
Comment (liewegas) [reply .github/stale.yml]: "stale" is an adjective (e.g., "stale bread"), not a verb (e.g., not "i staled the bread").

**Pull request merged**